### ISSUE TITLE
Error in manual installation manual_setup.ipynb

### ISSUE THREAD

**Issue by SilalahiDivo** [jeffheaton/t81_558_deep_learning#62]
Hi @jeffheaton , thanks for your awesome resources for deep learning. I a new user in GitHub and also new comer for deep learning. Now I am starting to subscribe and follow your youtube channel. Really this would help me during my work. 

I just followed your manual instruction for installation as in your GitHub manual_setup.ipynb. Unfortunately I have followed all the instruction as mentioned but I when I run the code in Jupiter:

```
# What version of Python do you have?
import sys

import tensorflow.keras
import pandas as pd
import sklearn as sk
import tensorflow as tf

print(f"Tensor Flow Version: {tf.__version__}")
print(f"Keras Version: {tensorflow.keras.__version__}")
print()
print(f"Python {sys.version}")
print(f"Pandas {pd.__version__}")
print(f"Scikit-Learn {sk.__version__}")
print("GPU is", "available" if tf.test.is_gpu_available() else "NOT AVAILABLE")


```

and the output are:

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
~\AppData\Roaming\Python\Python36\site-packages\tensorflow\python\pywrap_tensorflow.py in <module>
     57 
---> 58   from tensorflow.python.pywrap_tensorflow_internal import *
     59 

~\AppData\Roaming\Python\Python36\site-packages\tensorflow\python\pywrap_tensorflow_internal.py in <module>
     27             return _mod
---> 28     _pywrap_tensorflow_internal = swig_import_helper()
     29     del swig_import_helper

~\AppData\Roaming\Python\Python36\site-packages\tensorflow\python\pywrap_tensorflow_internal.py in swig_import_helper()
     23             try:
---> 24                 _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
     25             finally:

C:\ProgramData\Miniconda3\envs\py36\lib\imp.py in load_module(name, file, filename, details)
    242         else:
--> 243             return load_dynamic(name, filename, file)
    244     elif type_ == PKG_DIRECTORY:

C:\ProgramData\Miniconda3\envs\py36\lib\imp.py in load_dynamic(name, path, file)
    342             name=name, loader=loader, origin=path)
--> 343         return _load(spec)
    344 

ImportError: DLL load failed: The specified module could not be found.

During handling of the above exception, another exception occurred:

ImportError                               Traceback (most recent call last)
<ipython-input-1-812fc96d3476> in <module>
      2 import sys
      3 
----> 4 import tensorflow.keras
      5 import pandas as pd
      6 import sklearn as sk

~\AppData\Roaming\Python\Python36\site-packages\tensorflow\__init__.py in <module>
     39 import sys as _sys
     40 
---> 41 from tensorflow.python.tools import module_util as _module_util
     42 from tensorflow.python.util.lazy_loader import LazyLoader as _LazyLoader
     43 

~\AppData\Roaming\Python\Python36\site-packages\tensorflow\python\__init__.py in <module>
     48 import numpy as np
     49 
---> 50 from tensorflow.python import pywrap_tensorflow
     51 
     52 # Protocol buffers

~\AppData\Roaming\Python\Python36\site-packages\tensorflow\python\pywrap_tensorflow.py in <module>
     67 for some common reasons and solutions.  Include the entire stack trace
     68 above this error message when asking for help.""" % traceback.format_exc()
---> 69   raise ImportError(msg)
     70 
     71 # pylint: enable=wildcard-import,g-import-not-at-top,unused-import,line-too-long

ImportError: Traceback (most recent call last):
  File "C:\Users\ASDUser\AppData\Roaming\Python\Python36\site-packages\tensorflow\python\pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "C:\Users\ASDUser\AppData\Roaming\Python\Python36\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "C:\Users\ASDUser\AppData\Roaming\Python\Python36\site-packages\tensorflow\python\pywrap_tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
  File "C:\ProgramData\Miniconda3\envs\py36\lib\imp.py", line 243, in load_module
    return load_dynamic(name, filename, file)
  File "C:\ProgramData\Miniconda3\envs\py36\lib\imp.py", line 343, in load_dynamic
    return _load(spec)
ImportError: DLL load failed: The specified module could not be found.


Failed to load the native TensorFlow runtime.

See https://www.tensorflow.org/install/errors

for some common reasons and solutions.  Include the entire stack trace
above this error message when asking for help.
```

For your info I uses windows 7, with miniconda 3 (64-bit) and Python version 3.6.10

**Comment by SilalahiDivo** [comment]
UPDATE: Hi @jeffheaton I tried to unistall the tensorflow and reinstall it again as conda list below:

```
(py36) C:\Users\ASDUser>conda list
# packages in environment at C:\ProgramData\Miniconda3\envs\py36:
#
# Name                    Version                   Build  Channel
absl-py                   0.9.0                    pypi_0    pypi
astor                     0.8.1                    pypi_0    pypi
attrs                     19.3.0                     py_0
backcall                  0.1.0                    py36_0
bayesian-optimization     1.1.0                    pypi_0    pypi
blas                      1.0                         mkl
bleach                    3.1.4                      py_0
blis                      0.4.1                    pypi_0    pypi
boto                      2.49.0                   pypi_0    pypi
boto3                     1.13.11                  pypi_0    pypi
botocore                  1.16.11                  pypi_0    pypi
ca-certificates           2020.1.1                      0
catalogue                 1.0.0                    pypi_0    pypi
certifi                   2020.4.5.1               py36_0
chardet                   3.0.4                    pypi_0    pypi
click                     7.1.2                    pypi_0    pypi
cloudpickle               1.3.0                    pypi_0    pypi
colorama                  0.4.3                      py_0
cycler                    0.10.0                   pypi_0    pypi
cymem                     2.0.3                    pypi_0    pypi
cython                    0.29.14                  pypi_0    pypi
decorator                 4.4.2                      py_0
defusedxml                0.6.0                      py_0
docutils                  0.15.2                   pypi_0    pypi
entrypoints               0.3                      py36_0
flask                     1.1.2                    pypi_0    pypi
future                    0.18.2                   pypi_0    pypi
gast                      0.3.3                    pypi_0    pypi
gensim                    3.8.3                    pypi_0    pypi
google-pasta              0.2.0                    pypi_0    pypi
grpcio                    1.29.0                   pypi_0    pypi
gym                       0.17.2                   pypi_0    pypi
h5py                      2.10.0                   pypi_0    pypi
icc_rt                    2019.0.0             h0cc432a_1
icu                       58.2                 ha925a31_3
idna                      2.9                      pypi_0    pypi
importlib_metadata        1.5.0                    py36_0
intel-openmp              2020.1                      216
ipykernel                 5.1.4            py36h39e3cac_0
ipython                   7.13.0           py36h5ca1d4c_0
ipython_genutils          0.2.0                    py36_0
ipywidgets                7.5.1                      py_0
itsdangerous              1.1.0                    pypi_0    pypi
jedi                      0.17.0                   py36_0
jinja2                    2.11.2                     py_0
jmespath                  0.10.0                   pypi_0    pypi
joblib                    0.15.0                   pypi_0    pypi
jpeg                      9b                   hb83a4c4_2
jsonschema                3.2.0                    py36_0
jupyter                   1.0.0                    py36_7
jupyter_client            6.1.3                      py_0
jupyter_console           6.1.0                      py_0
jupyter_core              4.6.3                    py36_0
keras                     2.2.4                    pypi_0    pypi
keras-applications        1.0.8                    pypi_0    pypi
keras-preprocessing       1.1.2                    pypi_0    pypi
kiwisolver                1.2.0                    pypi_0    pypi
libpng                    1.6.37               h2a8f88b_0
libsodium                 1.0.16               h9d3ae62_0
lxml                      4.5.0                    pypi_0    pypi
m2w64-gcc-libgfortran     5.3.0                         6
m2w64-gcc-libs            5.3.0                         7
m2w64-gcc-libs-core       5.3.0                         7
m2w64-gmp                 6.1.0                         2
m2w64-libwinpthread-git   5.0.0.4634.697f757               2
markdown                  3.2.2                    pypi_0    pypi
markupsafe                1.1.1            py36he774522_0
matplotlib                3.2.1                    pypi_0    pypi
mistune                   0.8.4            py36he774522_0
mkl                       2020.1                      216
mkl-service               2.3.0            py36hb782905_0
mkl_fft                   1.0.15           py36h14836fe_0
mkl_random                1.1.0            py36h675688f_0
msys2-conda-epoch         20160418                      1
murmurhash                1.0.2                    pypi_0    pypi
nbconvert                 5.6.1                    py36_0
nbformat                  5.0.6                      py_0
notebook                  6.0.3                    py36_0
numpy                     1.18.1           py36h93ca92e_0
numpy-base                1.18.1           py36hc3f5095_1
openssl                   1.1.1g               he774522_0
pandas                    1.0.3                    pypi_0    pypi
pandas-datareader         0.8.1                    pypi_0    pypi
pandoc                    2.2.3.2                       0
pandocfilters             1.4.2                    py36_1
parso                     0.7.0                      py_0
pickleshare               0.7.5                    py36_0
pillow                    7.1.2                    pypi_0    pypi
pip                       20.0.2                   py36_3
plac                      1.1.3                    pypi_0    pypi
preshed                   3.0.2                    pypi_0    pypi
prometheus_client         0.7.1                      py_0
prompt-toolkit            3.0.4                      py_0
prompt_toolkit            3.0.4                         0
protobuf                  3.12.0                   pypi_0    pypi
pyglet                    1.5.0                    pypi_0    pypi
pygments                  2.6.1                      py_0
pyparsing                 2.4.7                    pypi_0    pypi
pyqt                      5.9.2            py36h6538335_2
pyrsistent                0.16.0           py36he774522_0
python                    3.6.10               h9f7ef89_2
python-dateutil           2.8.1                      py_0
pytz                      2020.1                   pypi_0    pypi
pywin32                   227              py36he774522_1
pywinpty                  0.5.7                    py36_0
pyyaml                    5.3.1                    pypi_0    pypi
pyzmq                     18.1.1           py36ha925a31_0
qt                        5.9.7            vc14h73c81de_0
qtconsole                 4.7.4                      py_0
qtpy                      1.9.0                      py_0
requests                  2.23.0                   pypi_0    pypi
s3transfer                0.3.3                    pypi_0    pypi
scikit-learn              0.23.0                   pypi_0    pypi
scipy                     1.4.1            py36h9439919_0
send2trash                1.5.0                    py36_0
setuptools                46.2.0                   py36_0
sip                       4.19.8           py36h6538335_0
six                       1.14.0                   py36_0
sklearn                   0.0                      pypi_0    pypi
smart-open                2.0.0                    pypi_0    pypi
spacy                     2.2.4                    pypi_0    pypi
sqlite                    3.31.1               h2a8f88b_1
srsly                     1.0.2                    pypi_0    pypi
tb-nightly                1.14.0a20190603          pypi_0    pypi
tensorboard               2.2.1                    pypi_0    pypi
tensorflow                2.0.0b1                  pypi_0    pypi
tensorflow-hub            0.8.0                    pypi_0    pypi
termcolor                 1.1.0                    pypi_0    pypi
terminado                 0.8.3                    py36_0
testpath                  0.4.4                      py_0
tf-estimator-nightly      1.14.0.dev2019060501          pypi_0    pypi
theano                    1.0.4                    pypi_0    pypi
thinc                     7.4.0                    pypi_0    pypi
threadpoolctl             2.0.0                    pypi_0    pypi
tornado                   6.0.4            py36he774522_1
tqdm                      4.46.0                   pypi_0    pypi
traitlets                 4.3.3                    py36_0
urllib3                   1.25.9                   pypi_0    pypi
vc                        14.1                 h0510ff6_4
vs2015_runtime            14.16.27012          hf0eaf9b_1
wasabi                    0.6.0                    pypi_0    pypi
wcwidth                   0.1.9                      py_0
webencodings              0.5.1                    py36_1
werkzeug                  1.0.1                    pypi_0    pypi
wheel                     0.34.2                   py36_0
widgetsnbextension        3.5.1                    py36_0
wincertstore              0.2              py36h7fe50ca_0
winpty                    0.4.3                         4
wrapt                     1.12.1                   pypi_0    pypi
zeromq                    4.3.1                h33f27b4_3
zipp                      3.1.0                      py_0
zlib                      1.2.11               h62dcd97_4

(py36) C:\Users\ASDUser>python
Python 3.6.10 |Anaconda, Inc.| (default, May  7 2020, 19:46:08) [MSC v.1916 64 b
it (AMD64)] on win32
```

With output similar as like in your modul, but there are still some red notice:

```


C:\ProgramData\Miniconda3\envs\py36\lib\site-packages\tensorflow\python\framework\dtypes.py:523: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
C:\ProgramData\Miniconda3\envs\py36\lib\site-packages\tensorflow\python\framework\dtypes.py:524: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
C:\ProgramData\Miniconda3\envs\py36\lib\site-packages\tensorflow\python\framework\dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
C:\ProgramData\Miniconda3\envs\py36\lib\site-packages\tensorflow\python\framework\dtypes.py:526: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
C:\ProgramData\Miniconda3\envs\py36\lib\site-packages\tensorflow\python\framework\dtypes.py:527: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
C:\ProgramData\Miniconda3\envs\py36\lib\site-packages\tensorflow\python\framework\dtypes.py:532: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
Using TensorFlow backend.





Tensor Flow Version: 1.12.0
Keras Version:2.2.4

Python 3.6.10 |Anaconda, Inc.| (default, May  7 2020, 19:46:08) [MSC v.1916 64 bit (AMD64)]
Pandas 1.0.3
Scikit-Learn 0.23.0
GPU is NOT AVAILABLE
```

Can you help me to tackle this issue @jeffheaton ..many many thanks

**Comment by SilalahiDivo** [comment]
Updated: I have solved the issue by uninstall the numpy 1.18.4 and install the lower version 1.16.4

```
pip uninstall numpy
pip install numpy==1.16.4
```
